app/Http/Controllers/CommentController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Comment;
+use App\Services\ModelLogger;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+
+class CommentController
+{
+    public function send($articleId, Request $request, ModelLogger $logger): RedirectResponse
+    {
+        // Implement request validation
+
+        $comment = new Comment();
+        $comment->article_id = $articleId;
+        $comment->author_email = $request['formCommentEmail'];
+        $comment->message = $request['formCommentComment'];
+        $comment->save();
+
+        $logger->logModel($request->user(), $comment);
```

Comment:
Sorry but creation of the comment is not a solution. I request the information reading from product or service. You can add simple page with just var_dump for it with temp route, to identify that it will work.

Reply:
Added logger for adding a product to order. Yet still have to implement order and basket functionality.